Automattic/wp-calypso
# Happy-blocks: Move universal-header to a PHP template

### Changes

This moves the Universal Header happy block from a JS block to a PHP template (not a block at all). This makes it usable in any theme and it's really straight forward.

### Testing
1. Check this branch out.
2. cd into `apps/happy-blocks`.
3. Run `yarn dev --sync`.
4. Sandbox wordpress.com.

Follow the testing steps of 381-gh-Automattic/wpsupport3.
